.gitignore

```diff
@@ -1,3 +1,6 @@
+# Database
+*.sqlite
+
```

Comment:
This is no longer needed if the database goes in the `.cache` directory

Reply:
A path can possibly change, an extension is fixed, let's keep it.